src/main/java/com/jfrog/ide/idea/configuration/GlobalSettings.java

```diff
@@ -217,4 +221,16 @@ private void migrateCredentialsFromFileToPasswordSafe(ServerConfigImpl serverCon
     private boolean shouldPerformCredentialsMigration(ServerConfigImpl xrayConfig) {
         return !isAnyBlank(xrayConfig.getUsername(), xrayConfig.getPassword());
     }
+
+    public void loadConnectionDetailsFromJfrogCli() {
```

Comment:
Let's add the following description to this method -

The plugin supports reading the JFrog connection details from JFrog CLI's configuration.
This allows developers who already have JFrog CLI installed and configured, to have IDEA load the config automatically.